libs/pdi-scanner/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "@votingworks/pdi-scanner",
+  "version": "1.0.0",
+  "private": true,
+  "description": "Scanner interface for the PDI PageScan 6",
+  "license": "AGPL-3.0",
+  "author": "VotingWorks Eng <eng@voting.works>",
+  "main": "build/index.js",
+  "types": "build/index.d.js",
+  "files": [
+    "build"
+  ],
+  "scripts": {
+    "build": "pnpm install:rust-addon && pnpm build:rust-addon && pnpm build:ts",
+    "build:ts": "tsc --build tsconfig.build.json",
+    "build:rust-addon": "cargo build --release",
```

Comment:
For trusted build purposes, we should add `--offline` to this build command, i.e. `cargo build --release --offline`